wiki/Welcome/en.md

```diff
@@ -1,11 +1,11 @@
 # Welcome
 
-Welcome to osu!, a free-to-win rhythm game developed by peppy with four game modes: osu!standard, a circle clicking simulator; osu!taiko, a drumming emulator; osu!catch, a fruit salad catcher; osu!mania, a key smashing mania.
+Welcome to osu!, a free-to-win rhythm game developed by peppy with four game modes: osu!standard, a circle clicking simulator; osu!taiko, a drumming emulator; osu!catch, a fruit salad catcher; and osu!mania, a key smashing synthesizer.
 
 ## Common points of interest
 
-- [Registration](/wiki/Registration)
 - [Installation](/wiki/Installation)
+- [Registration](/wiki/Registration)
 - [Rules](/wiki/Rules)
 - [FAQ](/wiki/FAQ)
 - Game Modes
```

Comment:
Link `Game Modes` to https://osu.ppy.sh/help/wiki/Game_Modes.

Reply:
The Game modes page itself, in its current state, isn't really interesting as it just links to those other pages